src/components/program/ProgramSubscriptionPlanAdminCard.tsx

```diff
@@ -69,7 +73,80 @@ const ProgramSubscriptionPlanAdminCard: React.FC<{
     ? 'subscription'
     : programPlan.periodAmount && programPlan.periodType
     ? 'period'
+    : programPlan.card_id
+    ? 'membership'
     : 'perpetual'
+
+  const RenderProgramPlan: React.FC = () => {
```

Comment:
This naming is a bit confusing
Could be discuss this on daily

Reply:
`ProgramPlanModal`

Reply:
https://www.notion.so/kkshyu/Code-Review-c29ef181f60d489e939869373e77a065?pvs=4#aa2e298692d7442086038bacdef2840b

Reply:
// python 
controllers.py: BaseController / MainController / ...
// typescript
utils.ts: convertCurrency / convertDate / ...
helpers.ts: convertCurrency / convertDate / ...

// folder
controllers/
    BaseController.py
    base_controller.py

// database table
1. member // kolable
2. members // hasura, sequalize